modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/StandardHeader.scala

```diff
@@ -0,0 +1,90 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.seqexec.model.dhs.ObsId
+
+import edu.gemini.seqexec.server.DhsClient._
+
+/**
+  * Created by jluhrs on 1/31/17.
+  */
+
+trait TcsKeywordsReader {
```

Comment:
Yes, but decided against it. The implementation of `get[A](name: String):A` would still need to switch between all the possible values of name, so it is not less code. And misspelling a name would be a run time error instead of a compile time error.